students/cfarris/lessons/lesson05/assignment/test_database.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env Python3
+
+'''
+Pytest for database.py
+'''
+from pymongo import MongoClient
+import pytest
+from database import read_csv, import_data, show_avail_products, show_rentals, check_redundancy
+
+
+coll_dict = {'customers': 'customers.csv',
+             'products': 'products.csv',
+             'rentals': 'rentals.csv'
+             }
+
+
+class MongoDBConnection(object):
+    """Connect to MongoDB"""
+    def __init__(self, host='127.0.0.1', port=27017):
+
+        self.host = host
+        self.port = port
+        self.connection = None
+
+    def __enter__(self):
+        self.connection = MongoClient(self.host, self.port)
+        return self
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        self.connection.close()
```

Comment:
You can just import this, you should never be typing stuff like this out more than once...